tests/plugins/inlinetoolbar/manual/kama.html

```diff
@@ -40,11 +40,11 @@
 
 	function instanceReadyListener( evt ) {
 		var image = this.editable().findOne( 'img' ),
-			panel = new CKEDITOR.ui.inlineToolbarView( this );
+			panel = new CKEDITOR.ui.inlineToolbar( this );
 
 		image.on( 'click', function() {
 			panel.create( this );
-			panel.parts.content.setHtml( '<p style="padding: 10px">Clicked element: ' + this.getName() + '</p>' );
+			panel._toolbar.parts.content.setHtml( '<p style="padding: 10px">Clicked element: ' + this.getName() + '</p>' );
```

Comment:
New samples should no longer use old API, it should register real buttons instead.

It would be nice to add for instance 2 buttons, and a rich combo (e.g. bold, font color and font size just for an example).

Reply:
Ofc this change should be propagated to other skin manual tests.